cli/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ _Released 03/1/2023 (PENDING)_
 **Bugfixes:**
 
 - Fixed an issue where cookies were being duplicated with the same hostname, but a prepended dot. Fixed an issue where cookies may not be expiring correctly. Fixes [#25174](https://github.com/cypress-io/cypress/issues/25174), [#25205](https://github.com/cypress-io/cypress/issues/25205) and [#25495](https://github.com/cypress-io/cypress/issues/25495).
+- Fixed an issue where cookies weren't being synced when the application was stable. Fixed in [#25855](https://github.com/cypress-io/cypress/pull/25855). Fixes [#25835](https://github.com/cypress-io/cypress/issues/25835).
```

Comment:
is this a user-facing fix or just a test fix? if its just for tests, this does't need an entry

Reply:
It is also user facing since the cookies will now [sync immediately if Cypress is stable](https://github.com/cypress-io/cypress/pull/25855/files#diff-f2ac49674a7eed42b161510aeba4d9f058eb364563e7dbcede2f13f7c0da43e6R43)

Reply:
What does that mean to users though?

Reply:
in certain cases cookies weren't synced back to the automation client when using cy.origin (or if a cross origin injection occurs) if the application is already stable. This is mainly problematic for setting cookies inside cy.origin after the page has loaded / is stable that the app couldnt otherwise set